tyche-update-helper.php

```diff
@@ -3,21 +3,38 @@
 Plugin Name: Tyche Update Helper
 Plugin URI: #
```

Comment:
Please replace the plugin URI from # to https://www.tychesoftwares.com/premium-woocommerce-plugins/